release-notes/kvm/v11.1.1.md

```diff
@@ -0,0 +1,5 @@
+## :zap: Giant Swarm Release 11.1.1 for KVM is now active for you! :zap:
```

Comment:
Should there be some release notes here?

Reply:
Looked up to https://github.com/giantswarm/releases/blob/master/release-notes/kvm/v11.1.0.md but maybe that one was a bit rushed. 

It has this nginx IC update only. Any suggestion for summary?

Reply:
Got it and yes I think it can be short. 

Process wise I think POs are responsible for the release notes. @cokiengchiara WDYT?

Reply:
I suggested sth here https://github.com/giantswarm/releases/pull/88/files#r375976415 pls provide feedback there :)

Reply:
Added same release overview as for aws 9.1.1 although we need to rephrase both since we know now that there are breaking changes https://github.com/giantswarm/giantswarm/issues/8851#issuecomment-583723774